invenio_rdm_records/services/config.py

```diff
@@ -41,66 +41,52 @@
 class RDMSearchOptions(SearchOptions):
     """Search options for record search."""
 
-    facets_options = dict(
-        aggs={
-            'resource_type': {
-                'terms': {'field': 'metadata.resource_type.type'},
-                'aggs': {
-                    'subtype': {
-                        'terms': {'field': 'metadata.resource_type.subtype'},
-                    }
-                }
-            },
-            'access_status': {
-                'terms': {'field': 'access.status'},
-            },
-            # 'languages': {
-            #     'terms': {'field': 'metadata.languages.id'},
-            # },
-        },
-        post_filters={
-            'resource_type': nested_terms_filter(
-                'metadata.resource_type.type',
-                'metadata.resource_type.subtype',
-                splitchar='::',
-            ),
-            'access_status': terms_filter('access.status'),
-            # 'languages': terms_filter('metadata.languages.id'),
-        }
-    )
+    facets = {
+        'resource_type': NestedTermsFacet(
+            field='metadata.resource_type.type',
+            subfield='metadata.resource_type.subtype',
+            label=_("Resource types"),
+            # value_labels=VocabularyLabels('resource_types')
```

Comment:
@ppanero This is the class you have to implement.

Reply:
Should I be touching this too when I tackle resource types?